components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIProviderHostObject.java

```diff
@@ -426,6 +434,104 @@ public static NativeArray jsFunction_getBusinessOwners(Context cx, Scriptable th
         return myn;
     }
 
+    /**
+     * Retrieve business owner.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files